com.jetbrains.Intellij-Community.json

```diff
@@ -0,0 +1,72 @@
+{
+    "app-id": "com.jetbrains.Intellij-Community",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk9"
+    ],
+    "command": "intellij-desktop",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=host",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.freedesktop.secrets",
+        "--filesystem=xdg-run/keyring",
+        "--env=PATH=/usr/bin:/app/bin:/usr/lib/sdk/openjdk9/bin"
+    ],
+    "modules": [
+          {
+              "name": "openjdk",
+              "buildsystem": "simple",
+              "build-commands": [
+                "/usr/lib/sdk/openjdk9/install.sh"
+              ]
+          },
+          {
+            "name": "intellij",
+            "buildsystem": "simple",
+            "build-commands": [
+                "tar xzfv ideaIC-2017.3.5.tar.gz",
+                "rm -f ideaIC-2017.3.5/idea-IC-173.4674.33/bin/fsnotifier*",
```

Comment:
```suggestion

```
Probably `fs-notifier` should be kept as explained in https://github.com/flathub/com.jetbrains.PyCharm-Community/pull/27